scripts/build-splunkrc.py

```diff
@@ -1,112 +0,0 @@
-# Copyright 2011-2020 Splunk, Inc.
-#
-# Licensed under the Apache License, Version 2.0 (the "License"): you may
-# not use this file except in compliance with the License. You may obtain
-# a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
-# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
-# License for the specific language governing permissions and limitations
-# under the License.
-
-#!/usr/bin/env python
-
-import sys
-import json
-import urllib.parse
-import os
-from pathlib import Path
-from string import Template
-
-DEFAULT_CONFIG = {
-    'host': 'localhost',
-    'port': '8089',
-    'username': 'admin',
-    'password': 'changed!',
-    'scheme': 'https',
-    'version': '8.0'
-}
-
-DEFAULT_SPLUNKRC_PATH = os.path.join(str(Path.home()), '.splunkrc')
```

Comment:
So we actually still need this script - it's used to convert an orca configuration into .splunkrc for a separate set of smoke tests that runs against the Splunk platform - so this actually needs to be altered rather than removed, output to `.env` in the appropriate format rather than `.splunkrc`. Can we rename this to `build-env.py` and output make the env format changes needed? Here's a sample command and input that we need to continue working:

Old:
```
$ cat orca.json | python3 scripts/build-splunkrc.py ~/.splunkrc
```
New:
```
$ cat orca.json | python3 scripts/build-env.py ~/.env
```
I'll slack you some sample contents for orca.json

Reply:
Renamed file to build-env.py and did some changes to create .env file from script
Tested with sample orca.json file and it is working as expected